ckanext/datitrentinoit/model/mapping.py

```diff
@@ -172,6 +172,16 @@ def create_pro_package_dict(guid, swpentry: StatWebProEntry, metadata: StatWebMe
     extras['_harvest_source'] = 'statistica:' + swpentry.get_id()
     extras["identifier"] = f'{DEFAULT_IPA}:ispat_{swpentry.get_id()}'
     extras['source_url'] = swpentry.get_url()
+    extras['creator'] = [{
+        "creator_identifier": "XGT4IE",
+        "creator_name": "ISPAT Istituto di statistica della provincia di Trento"
+    }]
+    extras['contact_point_name'] = "XGT4IE"
+    extras['contact_point_identifier'] = "ISPAT Istituto di statistica della provincia di Trento"
+    extras['contact_point'] = [{
+        "contact_point_identifier": "XGT4IE",
+        "contact_point_name": "ISPAT Istituto di statistica della provincia di Trento"
```

Comment:
Do we need contact point info to be replicated both in the extras and as a list?
By the doc contact point is 0..n, so the list should be the correct encoding.